store/sql_webhook_store.go

```diff
@@ -52,6 +54,14 @@ func NewSqlWebhookStore(sqlStore *SqlStore) WebhookStore {
 		tableo.ColMap("Description").SetMaxSize(128)
 		tableo.ColMap("ContentType").SetMaxSize(128)
 		tableo.ColMap("TriggerWhen").SetMaxSize(1)
+
+		tablec := db.AddTableWithName(model.CommandWebhook{}, "CommandWebhooks").SetKeys(false, "Id")
```

Comment:
Can you move the `CommandWebhooks` table into its own `sql_command_webhook_store.go` file? It doesn't really relate to the other webhooks enough to have it in the same file